### PR TITLE
Add Github CI

### LINKED ISSUE #12: Setup CI
- [ ] build
- [ ] lint
- [ ] gettext

### PR DESCRIPTION
**Pull request by yasharjit**
Fixes #12

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    container:
+      image: elementary/docker:unstable
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install Dependencies
+      run: |
+        apt update
+        apt install -y libfolks-dev libglib2.0-dev libgranite-dev libgtk-3-dev
+        apt install -y meson valac
```
Comment (ryonakano) [inline .github/workflows/main.yml:19]:
```suggestion
        apt install -y libfolks-dev
```

Actually the elementary docker installs fundamental dependencies like meson or valac, so what we need to explicit is only folks

Comment (yasharjit) [reply .github/workflows/main.yml:19]: are glib, granite and gtk needed or are they also installed by the elementary docker image?

Comment (ryonakano) [reply .github/workflows/main.yml:19]: The elementary docker image [installs `elementary-sdk`](https://github.com/elementary/docker/blob/master/odin-unstable/Dockerfile#L13) which [includes glib, granite, gtk, meson, and valac as dependencies](https://github.com/elementary/metapackages/blob/master/elementary-sdk-amd64).

Comment (yasharjit) [reply .github/workflows/main.yml]: Done.

**Review by ryonakano — APPROVED** [review]
Looks good, thank you!

**Pull request merged**